modules/update-conditional-updates.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-cli.adoc
+// * updating/understanding-upgrade-channels-releases.adoc
+
+[id="update-conditional-upgrade-path{context}"]
+= Updating to a conditional upgrade path
+
+You can update a conditional upgrade path using the OpenShift CLI (`oc`) 4.10 or later.
```

Comment:
Possibly worth pointing out again here that when a conditional update happens to be recommended for your cluster, that you can update using the same web console and `oc` commands that folks have been using since 4.1.  It's only when a conditional update is not recommended for your cluster that you want the 4.10+ `oc`.

Reply:
Yeah, agree. And if we pointing it out by wording, I think we can remove line#13 through line#21.